infrastructure-playbooks/rolling_update.yml

```diff
@@ -76,15 +79,11 @@
   become: True
 
   pre_tasks:
-    - include_vars: roles/ceph-common/defaults/main.yml
```

Comment:
Correct, however we currently have a bug.
Discovered by @guits yesterday, the `ansible.cfg` file is honoured on 2.2, see: https://github.com/ansible/ansible/issues/18090

We won't fix this with this commit, we just need to be aware of that and potentially offer a fix in a separate PR.